src/main/java/thePackmaster/cards/rimworldpack/EatWithoutTable.java

```diff
@@ -0,0 +1,63 @@
+package thePackmaster.cards.rimworldpack;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.animations.TalkAction;
+import com.megacrit.cardcrawl.actions.common.MakeTempCardInHandAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.helpers.PotionHelper;
+import com.megacrit.cardcrawl.localization.UIStrings;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.potions.AbstractPotion;
+import com.megacrit.cardcrawl.potions.SwiftPotion;
+import com.megacrit.cardcrawl.relics.AbstractRelic;
+import thePackmaster.cards.AbstractPackmasterCard;
+import thePackmaster.powers.rimworldpack.SpikeTrapPower;
+import thePackmaster.util.Wiz;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+import static thePackmaster.SpireAnniversary5Mod.modID;
+
+public class EatWithoutTable extends AbstractPackmasterCard {
+    public final static String ID = makeID(EatWithoutTable.class.getSimpleName());
+
+    public EatWithoutTable() {
+        super(ID, 1, CardType.SKILL, CardRarity.COMMON, CardTarget.SELF);
+        exhaust = true;
+        cardsToPreview = new Despair();
+    }
+
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        //Drink a random potion
+        addToBot(new AbstractGameAction() {
+            @Override
+            public void update() {
+                AbstractPotion potion = null;
+                int tries = 0;
+                while(tries < 100 && potion == null){
```

Comment:
This is pretty jank. Please just create a list of potions by iterating through PotionHelp.potions. Then take a random one from that and make a copy of it